### PR TITLE
Update 'Introduction to Matrix'.

### PR DESCRIPTION
**Pull request by q-b**
Small update with shameless advertisement for rust projects.

### CODE REVIEW

File: content/docs/matrix/_index.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Users can decide who runs their homeserver:
 * Yourself: You can run the homeserver software on your own computer.
 
 As a homeserver operator, you can choose your homeserver implementation.
-The team that developed Matrix provides an implementation called Synapse.
-Ruma is another option.
+The team that developed Matrix provides an implementation called [Synapse](https://matrix.org/docs/projects/server/synapse).
+[Conduit](conduit.rs) which uses Ruma is another option.
```
Comment (jplatte) [inline content/docs/matrix/_index.md]: I think we should note that it's experimental. Also link is incomplete.

```suggestion
[Conduit](https://conduit.rs/), which is built on Ruma, is another (experimental) option.
```

File: content/docs/matrix/_index.md [hunk 2]
```diff
@@ -83,13 +83,13 @@ The table shows the protocol each system uses, examples of resources you'd acces
...
       <td>sendmail, postfix</td>
-      <td>Ruma, Synapse</td>
+      <td>Conduit, Synapse, Dendrite</td>
     </tr>
     <tr>
       <th>You use…</th>
       <td>Chrome, Firefox</td>
       <td>Gmail web app, Apple Mail</td>
-      <td>Riot, WeeChat</td>
+      <td>Element, Fractal, Daydream</td>
```
Comment (jplatte) [inline content/docs/matrix/_index.md]: Not sure whether to mention Daydream (cc @MTRNord). In any case, WeeChat can still be used, there's even a rewrite of the Matrix integration [in Rust](https://github.com/poljar/weechat-matrix-rs).

```suggestion
      <td>Element, Fractal, WeeChat</td>
```

**Review by jplatte — COMMENTED** [review]
Thanks, this is long overdue :sweat_smile:

File: content/docs/matrix/_index.md (hunk 2 above)
Comment (MTRNord) [reply content/docs/matrix/_index.md]: Yeah probably daydream isnt something that should be mentioned at this point as development is paused (waiting on dependencies)

Comment (q-b) [reply content/docs/matrix/_index.md]: 😢

Comment (MTRNord) [reply content/docs/matrix/_index.md]: I plan to start trying to get druid version updated about next weekend ;) So there will be development in the future

**Review by jplatte — APPROVED** [review]

**Pull request merged**